### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/argo-cd/commit/97cfe010600d6889a857f9c1a0e0c0f53b006ad4#commitcomment-43340950

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/argo-cd/commit/3e7e1b810e3785a837f03881d6dd2ddcb2b2eae4

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I've signed the CLA and my build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

### CODE REVIEW

File: controller/appcontroller.go [hunk 1]
```diff
@@ -1332,7 +1332,7 @@ func (ctrl *ApplicationController) autoSync(app *appv1.Application, syncStatus *
 			}
 		}
 		if bAllNeedPrune {
-			message := fmt.Sprintf("Skipping sync attempt to %s: auto-sync will wipe out all resourses", desiredCommitSHA)
+			message := fmt.Sprintf("Skipping sync attempt to %s: auto-sync will wipe out all resources", desiredCommitSHA)
```
Comment (jsoref) [inline controller/appcontroller.go:1335]: This is user facing -- specifically, we hit it which is why I'm running a pass so soon after my previous pass.

File: server/application/application_test.go [hunk 2]
```diff
@@ -152,13 +152,13 @@ func newTestAppServer(objects ...runtime.Object) *Server {
...
 	if !k8scache.WaitForCacheSync(ctx.Done(), appInformer.HasSynced) {
-		panic("Timed out waiting forfff caches to sync")
+		panic("Timed out waiting for caches to sync")
 	}
 
 	projInformer := factory.Argoproj().V1alpha1().AppProjects().Informer()
 	go projInformer.Run(ctx.Done())
 	if !k8scache.WaitForCacheSync(ctx.Done(), projInformer.HasSynced) {
-		panic("Timed out waiting forfff caches to sync")
+		panic("Timed out waiting for caches to sync")
```
Comment (jsoref) [inline server/application/application_test.go:161]: This is odd

**Review by wtam2018 — COMMENTED** [review]
LGTM

File: server/application/application_test.go (hunk 2 above)
Comment (alexmt) [reply server/application/application_test.go:161]: someone spilled coffee on keyboard :)

**Review by alexmt — APPROVED** [review]
LGTM

**Pull request merged**